_data/software/zint-barcode-generator.yml

```diff
@@ -0,0 +1,36 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: A barcode encoding library supporting over 50 symbologies.
+    fr: >-
+      Une bibliothèque d'encodage de codes à barres prenant en charge plus de 50
+      symbologies.
+category: '43233400'
+homepageURL:
+  en: 'https://zint.github.io/'
+  fr: 'https://zint.github.io/'
+licences:
+  - URL:
+      en: 'https://github.com/zint/zint/blob/master/COPYING'
```

Comment:
Looks like the active development is over on sourceforge, but this file is still the same date on both https://sourceforge.net/p/zint/code/ci/master/tree/COPYING